Deploy/Systemd/ss-update-daemon.service

```diff
@@ -6,6 +6,7 @@ After=network-online.target
 [Service]
 Type=oneshot
 ExecStart=/home/pi/SUSI.AI/susi_linux/update_daemon/update_check.sh
+RemainAfterExit=true
```

Comment:
Why?

Reply:
@hongquan , this came in as a mistake. I'll remove it